packages/file_selector/file_selector_linux/pubspec.yaml

```diff
@@ -2,7 +2,11 @@ name: file_selector_linux
 description: Liunx implementation of the file_selector plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/file_selector/file_selector_linux
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+file_selector%22
-version: 0.9.0+1
+version: 0.9.0+2
+
+# TODO(adpinola): remove this once file_selector_platform_interface version is updated to 2.3.0
+# with getDirectoriesPaths method.
+publish_to: none
```

Comment:
Please don't disable checks by marking things as non-published; the failures from the temporary use of paths are good, and should not be bypassed.